package.json

```diff
@@ -1,7 +1,7 @@
 {
   "private": true,
   "scripts": {
-    "install": "lerna bootstrap --hoist",
+    "install": "npm run update:core && lerna bootstrap --hoist",
```

Comment:
Right, I did that because Travis' Ubuntu image had some strange problem with `node-gyp` rebuild errors. It's not a huge deal, though.